test/test.py

```diff
@@ -496,3 +498,10 @@ def copy_source_data(self, target, source):
             "cp", "--reflink=auto", "-a",
             os.path.join(from_path, "."), to_path
         ], check=True)
+
+
+@pytest.fixture(name="osb")
+def osbuild_fixture():
```

Comment:
This is a slowdown for test_noop.py now - that used to not use"cache_from" and now it does which means a lot of cache is copied which is not neeed for the noop test and will slow the test down.